### ISSUE TITLE
Exif information under images

### ISSUE THREAD

**Issue by tattoodgeek** [worseisbetter/galleria#432]
I've been using galleriajs to put together an internal photo gallery at work for customer photos.  Everything is working fine except this last bit of showing EXIF information for images.  I was able to get it to show exif information for the first image in the gallery BUT when going through photos using the navigation arrows or carousel, the exif information is still of the first image.  Not sure if there is a way to get the page to refresh per image or to get the exif information to show per image being displayed.

Thank you.

**Comment by sammik** [comment]
As I know, galleria.js doesnt read EXIF data. You use some extern library for that?
Once you have data, you need, you can use it in galleria https://galleriajs.github.io/docs/references/data.html

**Comment by tattoodgeek** [comment]
no external library, just used some php to pull the info and added the below under the main vid but above the gallery_list div:

```
        </div>
		<div class="exif" id="exif">
			<!-- pulling exif information and displaying under image -->
				<?php
					$exif = exif_read_data(($dir.'/'.$pic['big']), 0, true);{
						echo "File Name: ".$exif['FILE']['FileName']."<br>";
						echo "Date/Time Taken: ".$exif['EXIF']['DateTimeOriginal']."<br>";
					}
				?>
		</div>
```

It is pulling the information I want to use BUT only for the first image of the gallery.  If images are changed via the carousel, it doesn't seem to be reloading any information thus not loading the EXIF info from that image.

**Comment by sammik** [comment]
This doesnt tell me much. Galleria is pure frontend.

**Comment by tattoodgeek** [comment]
Yes but this is what I added to the backend js to pull exif through php.
